x-pack/plugins/ml/public/data_frame/components/aggregation_list/list_summary.tsx

```diff
@@ -20,7 +20,9 @@ export const AggListSummary: React.SFC<AggListSummaryProps> = ({ list }) => {
     <EuiForm>
       {aggNames.map((aggName: AggName) => (
         <Fragment key={aggName}>
-          <EuiPanel paddingSize="s">{aggName}</EuiPanel>
+          <EuiPanel paddingSize="s">
+            <div className="mlAggregationLabel__text">{aggName}</div>
```

Comment:
Can you just use `eui-textTruncate` here?

Reply:
Good catch, didn't know about `eui-textTruncate`. Fixed in 0867f89.